modules/nf-core/stitch/main.nf

```diff
@@ -8,8 +8,8 @@ process STITCH {
         'biocontainers/r-stitch:1.6.10--r43h06b5641_0' }"
 
     input:
-    tuple val(meta) , path(posfile), path(input, stageAs: "input"), path(rdata, stageAs: "RData_in"), val(chromosome_name), val(K), val(nGen)
-    tuple val(meta2), path(collected_crams), path(collected_crais), path(cramlist)
+    tuple val(meta), path(collected_crams), path(collected_crais), path(cramlist)
```

Comment:
Why the swap?

Reply:
It was to be consistent with other imputation modules.
The input / target files first then the reference panel.

Reply:
Agreed then 👍🏻